ui/src/utils/createStackdriverUrl.js

```diff
@@ -18,20 +18,19 @@ const querystring = require("querystring");
 
 const stackdriverAPI = "https://console.cloud.google.com/logs/viewer";
 
-const stackdriverFilter = container => {
-  return `resource.type:"k8s_container"
-resource.labels.project_id:${container.gcp_project}
-resource.labels.cluster_name:${container.cluster}
-resource.labels.namespace_name:${container.namespace}
-resource.labels.pod_name:${container.pod_name}
-resource.labels.container_name:${container.name}`;
+const stackdriverFilter = query => {
+  return `resource.type:"k8s_query" OR "k8s_container" OR "k8s_pod"
+resource.labels.project_id:${query.gcp_project}
+resource.labels.cluster_name:${query.cluster}
+resource.labels.namespace_name:${query.namespace}
+resource.labels.pod_name:${query.pod_name}`;
```

Comment:
`resource.labels.pod_name:${query.pod_name}` --> this will restrict the query to only view one pod.  I think we should create query that is similar to the merlin logs (multiple pods)

Reply:
The pod_name value is something like this: `pod_a OR pod_b OR pod_c`. It allows stackdriver to stream logs from multiple pods.
The UI generate this value from Get Containers API: https://github.com/gojek/merlin/blob/improve-logging-ux/ui/src/components/logs/ContainerLogsView.js#L132-L142

Reply:
Ah ok got it.